replica/distributed_loader.cc

```diff
@@ -301,6 +301,9 @@ class table_populator {
     future<> start() {
         SCYLLA_ASSERT(this_shard_id() == 0);
```

Comment:
> sstable_directory: create_pending_deletion_log: place pending_delete log under the base directory

How does this work with downgrades?

1. upgrade a node
2. it crashes in a loop
3. downgrade
4. log is not detected

Reply:
Ah, the base directory log will be processed, and won't notice the files have an extra component. I think it's safe.

Reply:
exactly